src/testing/packet_simulator.zig

```diff
@@ -64,8 +64,11 @@ pub const PartitionMode = enum {
     /// equal-size partitions.
     uniform_partition,
 
-    /// Isolates exactly one replica.
+    /// Isolates exactly one replica (symmetric).
     isolate_single,
+
+    /// Isolates exactly one replica (asymmetric: send-only).
+    isolate_single_sender,
 };
```

Comment:
assymetric partitons make sense for every mode except `none`, so perhaps we a separate `assymetric_partitions: bool` param?